client/src/components/List/reducers.js

```diff
@@ -0,0 +1,22 @@
+import { combineReducers } from 'redux';
```

Comment:
If you're going to move the event reducer into this file, you should take out the combineReducers and just export the reducer.

We'll want to combine all of the reducers in one file outside of the component scope.